Automattic/wp-calypso
# composite-checkout: move analytics to own file

#### Changes proposed in this Pull Request

This change moves the calypso analytics function for composite checkout, which has started to become quite large, into its own file.

#### Testing instructions

- Visit checkout and make sure you see composite checkout (you can use the query string `?flags=composite-checkout-wpcom` if your user is not in the test group).
- Enable debugging by putting this in your JS console: `localStorage.setItem('debug', 'calypso:analytics')`
- Reload the page.
- Use the checkout form verify that you see events like `calypso_checkout_composite_step_changed`, `calypso_checkout_composite_first_step_complete`, etc.
- Verify that checkout works as expected.
